src/app/components/challenge-create/challenge-create.component.ts

```diff
@@ -73,33 +83,51 @@ export class ChallengeCreateComponent implements OnInit {
         setTimeout(() => {
           this.globalService.showToast('info', 'Please select a host team');
         }, 1000);
-        this.router.navigate(['/teams/hosts']);
+        this.router.navigate([this.hostTeamsRoute]);
       }
     });
   }
 
-  /**
-   * Form Validate function.
-   */
-  formValidate(formname) {
-    this.globalService.formValidate(this.components, this.formSubmit, this);
+  challengeCreate() {
+    if (this.ChallengeCreateForm['input_file'] !== null) {
+      const FORM_DATA: FormData = new FormData();
+      FORM_DATA.append('status', 'submitting');
+      FORM_DATA.append('zip_configuration', this.ChallengeCreateForm['input_file']);
+      this.globalService.startLoader('Creating Challenge');
+      this.challengeService.challengeCreate(
+        this.hostTeam['id'],
+        FORM_DATA,
+      ).subscribe(
+        data => {
+          this.globalService.stopLoader();
+          this.globalService.showToast('success', 'Successfuly sent to EvalAI admin for approval.');
+          this.router.navigate([this.hostedChallengesRoute]);
+        },
+        err => {
+          this.globalService.stopLoader();
+          this.globalService.showToast('error', err.error.error);
+          this.isSyntaxErrorInYamlFile = true;
+          this.syntaxErrorInYamlFile = err.error.error;
+        },
+        () => {}
+        );
+    } else {
+      this.isFormError = true;
+      this.globalService.showToast('error', 'Please Upload File');
```

Comment:
The position of this is not aligned correctly. Render it directly below the `Upload file` text field.